README.md

```diff
@@ -1 +1,196 @@
-# Leaderboard
\ No newline at end of file
+## _PREVIEW_
+<center><img src="https://lh3.googleusercontent.com/fife/AMPSemdJ9LDEILLap6mK5hmwnR1WxBBS6bROMl3xbyMSpf8kzUzeyKnt5evcfT9KzCZlCUETb8GlqqOsrcAOMucaKwj1TRcGqx2wyrlIxKvyxr_4OC56K2U6Uo_n3yp1dqtf8j2ZSkcJ2khnoHTFZFbEp9y98ThkCkB9YYwdQpyg8duz20OydPKEZ0KMdSbmT_9X5PqfYPyb9XlmVw01HbodLhLH7-oQFKGC7qrtY1kt4SsuBvpez7smHw-8UM5VO7rupUSTfjpkoFhy69qQDfWE3LBiXANcr9UTtCF5Gf1-iqbhXYiApmUrvwK6NK6x8wbmgw8XgC19hKF4kPUiDovtUVNWhDv2z9SOJbVY_g-PIBNdr02InAFIwl7B_33_540eaOZXSVLBSr_5B2k89CS00HH0qqNjJwbIFtxcH8l4FBRF41LuuuS6K7ptmsWvcow9D08h42LFS7x7pZ8rntmscl8wM-LAlBnzzWKM7Ac0xld-LzUqfa0EA-N9bLR_Ak3Z6srwEqGbwTvD0HS9jO-PTU7LojuUIyqWolIZg9OD_LvhqVSL7t4iUmnZm-SfRnkTfTmhBEc-KbbOIt458PQ-wH2Dw4EYNMpHpKq0gG6ZJR-Ekpd_OoTrcommBRTVJDN3O5MMHZkzcXkR46MkGdfA8Svqz6H_8W6MsC6_3U6Zz8c_1PupicUlJH1foLZLmX0sfGQZgSRicEXHIABnZfAbt78LWE9xn_xtgUCCuiR-iULXaJnsGvoox6yOLHYLu4qt_1DpiRZlMt-XCA-gK_xFZKcOicLbpLMF_4mVv3i7LitlO51dCfepsGwIOk_sLIiOE5qWLy8SCAI6gZ_pOKCp9hzE0XATuasddxb7DTfsHUCjIxwGWM9O6BdscRTe5bNqlkynl65TtPWjCXR6iCdbwVK_DbnXoauPUABkuOvpyEICGHoF-HaDvJ4DEwp0Hg5EQ4XFr3B9EvJbjvjlb-8MWPWrXznM-Z_iQ3gkGrx71GDaXW0imOPbhaHIcklT7H26B2ugUzBwZ7RIrWRY3bz6XXn8UZYATVHL8cRAiy0JsFGnTjc5HmfkOkvAWtAOpdXUNjI8DjGVow_CodjDQ8ErNE9wKpzXujV1KnQeBeSNlNngxv4b6ujwgW4yphTAmwhW4oA-cTSMSrFmwCUwERLBc4PMYgUPzosvBSNLE9SwYsOuYtYUqTLBDfrOQydcEQHh4aNnWJneun3JXjSoMwkENvYKFoi12t54Ji6mWcoNLmjz-aswiOl5jt02SBqMFiYLz5LB02qnYWQT1FAbc6jZUKApjBL3kbp3uawKLSNKVLRKZpiCazQYmh7BKvr4xYiKPpIEj0_w4L-rYNPEs16jvnnF-QtslP0r9nSHtP4Pxy-znQtWyGETUqnqtmTbKKElk9GG4A3jkSpRdOHfOHHzpcOhMDX_-YEo1rhDlC2ISmpJdjZRz3bB5tgE9f47jGUUPW4flpr-x7adE2u-rHQ8CQ=w1366-h617"></center>
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+- Authors
+- Future Features- 
+ - Mobile version design
+- Desktop version design
+- Dynamic HTML for featured-speakers
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ OPTIONAL] To keep your readme professional, you should always delete commented-out code like this since it no longer servers a purpose after you have filled in your readme.